ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/JsonRpcHttpService.java

```diff
@@ -455,7 +455,7 @@ private JsonRpcResponse process(final JsonObject requestJson, final Optional<Use
         LOG.debug(e);
         return errorResponse(id, JsonRpcError.INVALID_PARAMS);
       } catch (final RuntimeException e) {
-        LOG.debug(e);
+        LOG.error(e);
```

Comment:
This will lose the stack trace because it resolves to LOG.error(Object) which converts the param to a String with toString().  
```suggestion
        LOG.error("Error processing JSON-RPC request", e);
```